src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -347,7 +460,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     window.open('tel:' + phone, '_blank');
   }
 
-  private _emailUser(e: Event) {
```

Comment:
should these also be protected so they can be called in extended components?

Reply:
Good point! Fixed